### PR TITLE
adding lstm files

### PR DESCRIPTION
**Pull request by nikhilp93**

### CODE REVIEW

**Comment by ohpyupi** [comment]
Let's remove the Mouse Dynamics repo from your PR. It is already in the main repo as a submodule.

**Comment by nikhilp93** [comment]
> Let's remove the Mouse Dynamics repo from your PR. It is already in the main repo as a submodule.

done. removed. added new commit to PR.

**Comment by ohpyupi** [comment]
Okay. Can we have a document like this (https://github.com/sjsu-cybersecurity/ml-based-mouse-authentication#mouse-authentication-using-cnn-model)

There please provide user commands so that anyone can download this repo and run the command and run a model.

**Comment by nikhilp93** [comment]
> Okay. Can we have a document like this (https://github.com/sjsu-cybersecurity/ml-based-mouse-authentication#mouse-authentication-using-cnn-model)
> 
> There please provide user commands so that anyone can download this repo and run the command and run a model.

done, added read me file.

File: lstm-files/airline-passengers.csv [hunk 1]
```diff
@@ -0,0 +1,145 @@
+"Month","Passengers"
```
Comment (ohpyupi) [inline lstm-files/airline-passengers.csv]: Do we need those CSV files?

File: lstm-files/shampoo.csv [hunk 2]
```diff
@@ -0,0 +1,37 @@
+"Month","Sales"
```
Comment (ohpyupi) [inline lstm-files/shampoo.csv]: Do we also need this csv file?

**Review by ohpyupi — APPROVED** [review]

**Pull request merged**